src/App.js

```diff
@@ -2,6 +2,8 @@ import * as React from 'react';
 import {Button, View} from 'react-native';
 import {createDrawerNavigator} from '@react-navigation/drawer';
 import {NavigationContainer} from '@react-navigation/native';
+
+import ProgressBar from './utils/progressbar';
```

Comment:
UI component imports should be near the end of the imports, and add a space on top